### DISCUSSION TITLE
steam eu4(欧陆风云4)1.29 大清王朝 汉化mod

### DISCUSSION THREAD

**Discussion by punkmonday** [Gamuxorg/bbs#215] [建议和闲聊]
游戏名称：欧陆风云4)
游戏名称(英文)：Europa Universalis IV
steam平台最近出了一个eu4 manzhu的清王朝版，很想玩玩，看看历史进程，9.21之前28元入手了，但是发现创意工坊只有一个支持windows的汉化包，看了下是替换dll文件，linux应该也可以，看看能不能一起汉化下？

**Comment by shouhuanxiaoji** [comment]
可以的，能不能把win的汉化包贴一下，会下载下来看看

**Comment by punkmonday** [comment]
> 可以的，能不能把win的汉化包贴一下，会下载下来看看

https://steamcn.com/t452777-1-1

**Comment by shouhuanxiaoji** [comment]
这游戏汉化 似乎还得先解决双字节问题。头疼

**Comment by wangling12** [comment]
> 这游戏汉化 似乎还得先解决双字节问题。头疼

很难，之前研究过，找到的双字节补丁全是win下面的，好像是从d3d9.dll里面做了文章

**Comment by MengxingLiu** [comment]
有没有新的进展呀？最近想玩　风云千年的mod，可是里面的中文文档也有转码的问题，不知所措

**Comment by ghost** [comment]
建议 直接wine吧
